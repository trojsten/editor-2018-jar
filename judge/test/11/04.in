STR:
General
ZI
STR:
Plukovnik
DOLYAC
STR:
Podplukovnik
ZY
STR:
Strateg
SMPWCGJ
STR:
Spion
X
STR:
Programator
YPSAAMV
STR:
Velmajstermaxisupergeneralveduci
GKGZRB
STR:
Nadpodpredzadplukovnikazarovendokoncaajgeneral
WDXDGAUH
