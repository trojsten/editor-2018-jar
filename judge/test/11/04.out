{"Velmajstermaxisupergeneralveduci": "WDXDGAUH", "Spion": "YPSAAMV", "Nadpodpredzadplukovnikazarovendokoncaajgeneral": "GKGZRB", "Programator": "DOLYAC", "General": "SMPWCGJ", "Strateg": "X", "Podplukovnik": "ZI", "Plukovnik": "ZY"}